src/GovUk.Education.ExploreEducationStatistics.Common/BlobContainers.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace GovUk.Education.ExploreEducationStatistics.Common
+{
+    public interface IBlobContainer
+    {
+        public string Name { get; }
+        public string EmulatedName { get; }
+    }
+
+    public static class BlobContainers
+    {
+        public static readonly BlobContainer PrivateReleaseFiles = new BlobContainer("releases");
+        public static readonly BlobContainer PublicReleaseFiles = new BlobContainer("downloads");
```

Comment:
Nice approach with all this stuff. Do we have any plans to make the release folder names consistent as well?

Reply:
Yes there is but there's a reason why I didn't do it on this PR. I've written it up in https://dfedigital.atlassian.net/browse/EES-1937 along with some other naming problems!